rotkehlchen/chain/evm/decoding/curve/decoder.py

```diff
@@ -559,6 +543,21 @@ def _handle_post_decoding(
                 )
                 break  # We only need to handle zap contracts once
 
+        for event in decoded_events:
```

Comment:
Couldn't this post decoding rule be switched to action items and thus be more performant?